src/core/blueprint/into_schema.rs

```diff
@@ -88,10 +121,11 @@ fn to_type(def: &Definition) -> dynamic::Type {
                     dyn_schema_field = dyn_schema_field.description(description);
                 }
                 for arg in field.args.iter() {
-                    dyn_schema_field = dyn_schema_field.argument(dynamic::InputValue::new(
-                        arg.name.clone(),
-                        to_type_ref(&arg.of_type),
-                    ));
+                    let input_value =
+                        dynamic::InputValue::new(arg.name.clone(), to_type_ref(&arg.of_type));
+                    let input_value =
+                        insert_serde_value_to_input_value(input_value, arg.default_value.clone());
+                    dyn_schema_field = dyn_schema_field.argument(input_value);
```

Comment:
```rust

                for arg in field.args.iter() {
                    let iv = dynamic::InputValue::new(
                        arg.name.clone(),
                        to_type_ref(&arg.of_type),
                    );
                    let iv = iv.default_value(arg.default_value.as_ref().map(|v | ConstValue::from_json(v.clone()).ok()).flatten().unwrap()); // TODO: handle unwrap
                    dyn_schema_field = dyn_schema_field.argument(iv);
                }
```

I think with v7.0.6 we can just pass the default value and everything else should work as is

Reply:
> ```rust
>                 for arg in field.args.iter() {
>                     let iv = dynamic::InputValue::new(
>                         arg.name.clone(),
>                         to_type_ref(&arg.of_type),
>                     );
>                     let iv = iv.default_value(arg.default_value.as_ref().map(|v | ConstValue::from_json(v.clone()).ok()).flatten().unwrap()); // TODO: handle unwrap
>                     dyn_schema_field = dyn_schema_field.argument(iv);
>                 }
> ```
> 
> I think with v7.0.6 we can just pass the default value and everything else should work as is

I am doing this already inside the `insert_serde_value_to_input_value` function but its not working for default value in input types